assets/trix/stylesheets/content.scss

```diff
@@ -1,4 +1,9 @@
 .trix-content {
+  h1 {
+    font-size: 1.6em;
```

Comment:
I think we should bump this font-size down to `1.2em`. That'll help make these feel more like subheadings in the message/doc rather than competing with the title of the message/doc.